downstream/modules/catalog/proc-preparing-worker-installer.adoc

```diff
@@ -36,38 +36,38 @@ h| servicescatalog_no_proxy |
 
 . Navigate to the {PlatformNameShort} setup or setup bundle installer directory.
```

Comment:
According to IBM style guide, use "set up" as two words when using it as a verb. 
Suggestion: Navigate to the {PlatformNameShort} setup or setup bundle installer directory.  -> Navigate to the {PlatformNameShort} setup or set up the bundle installer directory.

Reply:
hey @lmalivert 'setup' and 'setup bundle' are the names of the AAP  installers which is why they appear as they do here.